influx/databases.go

```diff
@@ -202,3 +197,34 @@ func (c *Client) showRetentionPolicies(ctx context.Context, name string) ([]chro
 
 	return results.RetentionPolicies(), nil
 }
+
+func (c *Client) showMeasurements(ctx context.Context, name string, limit, offset int) ([]chronograf.Measurement, error) {
```

Comment:
is `limit` untyped to allow for it to be `nil`?

Reply:
You've probably already discussed with @goller -- just typing this for the record: This is golang shorthand for defining both `limit` and `offset` as type `int`.